crates/event-streaming/proto/walrus/event/v1alpha/service.proto

```diff
@@ -0,0 +1,164 @@
+syntax = "proto3";
+
+package walrus.event.v1alpha;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+import "sui/rpc/v2beta/checkpoint_contents.proto";
+import "sui/rpc/v2beta/checkpoint_summary.proto";
+import "sui/rpc/v2beta/event.proto";
+
+// EventStreamingService provides a verifiable stream of events from the Sui
+// blockchain for an event stream. It allows clients to receive events
+// along with cryptographic proofs that tie them to a specific checkpoint.
+// It is designed to be forward-compatible, allowing for proof mechanisms
+// to be upgraded over time without breaking clients.
+service EventStreamingService {
```

Comment:
One API call that's missing is the one that fetches validator keys. But perhaps it belongs in a more generic `LightClientService`? If we do decide to create such a separate service, I feel like the `GetNonInclusionProof` call also belongs there.

Reply:
Validator keys are in end of epoch checkpoint (afaik). If in future we have a dedicated light client service, we can remove this endpoint from here, but right now we need this to be available in the local sidecar service, wdyt?